test/unittests/mongoBackend/mongoGetSubscriptions_test.cpp

```diff
@@ -96,12 +96,12 @@ static void prepareDatabaseV1Subs(void) {
                                                        "type" << "ONCHANGE" <<
                                                        "value" << BSON_ARRAY("AX2" << "AY2")
                                                        )) <<
-                        "throttling" << 5.0
+                        "throttling" << (long long) 5
                         );
 
     BSONObj sub3 = BSON("_id" << OID(SUB_OID3) <<
-                        "expiration" << 20000000 <<
-                        "lastNotification" << 25000000 <<
+                        "expiration" << (long long) 20000000 <<
+                        "lastNotification" << (long long) 25000000 <<
```

Comment:
We applied the approach "read both long/int, write always long" to lastNotifcation. Thus, I don't understand why 25000000 (as int) doesn't work... I think it should be researched, this could be the tip of an iceberg of a more important bug.